driver-core/src/main/com/mongodb/operation/AggregateToCollectionOperation.java

```diff
@@ -110,13 +141,30 @@ public AggregateToCollectionOperation(final MongoNamespace namespace, final List
      * @param pipeline the aggregation pipeline.
      * @param writeConcern the write concern to apply
      * @param aggregationLevel the aggregation level
-     * @since 3.10
+     * @since 3.11
      */
     public AggregateToCollectionOperation(final MongoNamespace namespace, final List<BsonDocument> pipeline,
```

Comment:
Intellij says this is never used - is it from 3.10? (its not clear from the diff) if it is - we should keep it otherwise we don't need it.